Makefile

```
@@ -41,6 +43,13 @@ test-coveralls: test-dependencies
 	@$(PHPUNIT) --coverage-clover build/logs/clover.xml
 	php vendor/bin/php-coveralls -v
 
+check-coding-style:
+	wget https://squizlabs.github.io/PHP_CodeSniffer/phpcs.phar -O $(PHPCS_FILENAME)
```

We need the rule to create PHPCS_FILENAME, and use that rule as a dependency in places we PHP-CS.

```suggestion
$(PHPCS_FILENAME):
	wget https://squizlabs.github.io/PHP_CodeSniffer/phpcs.phar -O $(PHPCS_FILENAME)
	
check-coding-style: $(PHPCS_FILENAME)
```